synapse/api/constants.py

```diff
@@ -183,6 +183,11 @@ class EventContentFields:
     MSC1772_ROOM_TYPE = "org.matrix.msc1772.type"
 
 
+class RoomTypes:
+    SPACE = "m.space"
+    MSC1772_SPACE = "org.matrix.msc1772.space"
```

Comment:
Note that #10161 removes support for the unstable identifiers, so depending on when these land, will probably want to remove this first.